napari/utils/settings/__init__.py

```diff
@@ -1,7 +1,139 @@
-"""Settings management.
-"""
+from contextvars import ContextVar
+from pathlib import Path
+from typing import Any, Optional, Union, cast
 
-from ._defaults import CORE_SETTINGS
-from ._manager import SETTINGS, get_settings
+from ...utils.translations import trans
+from ._napari_settings import NapariSettings
 
-__all__ = ['CORE_SETTINGS', 'get_settings', 'SETTINGS']
+__all__ = ['NapariSettings', 'get_settings']
+
+
+class _SettingsProxy:
+    """Backwards compatibility layer."""
+
+    def __getattribute__(self, name) -> Any:
+        import warnings
+
+        warnings.warn(
+            trans._(
+                "Accessing SETTINGS is deprecated. Please use get_settings()",
+                deferred=True,
+            ),
+            FutureWarning,
+        )
+        return getattr(get_settings(), name)
+
+
+# can we deprecate this?
+SETTINGS = _SettingsProxy()
```

Comment:
We should! but this would be breaking now... (even if new and mostly unused by users?)